### ISSUE TITLE
Should Fargate resources be rounded up?

### ISSUE THREAD

**Issue by mdneuzerling** [Netflix/metaflow#749]
If I'm using a Fargate compute environment with a `@batch(cpu = 1, memory = 4000)` decorator, I get the following error:

```
An error occurred (ClientException) when calling the SubmitJob operation:
    Fargate resource requirements (1.00 vCPU, 4000 MiB) not valid.
```

This is because [Fargate supports only specific configurations](https://aws.amazon.com/fargate/pricing/). If I were to use `memory = 4096` it would work just fine.

Should these values be automatically rounded up to the minimum configuration which meets or exceeds the requested CPU/memory limits? This would make Metaflow behaviour for Fargates more consistent with that for EC2 instances. Or should this be left alone since it's normal AWS behaviour?

**Comment by leeyh20** [comment]
Agree, I was trying to run the helloaws.py example inside the metaflow-tutorials, and i thought i would be able to run it using fargate

![image](https://user-images.githubusercontent.com/7023320/204221587-0852e2bd-7d96-46ff-8ec6-7ca6d15c35ed.png)

But then you would get an error:

![image](https://user-images.githubusercontent.com/7023320/204221665-90b794b8-54e2-443b-9bc1-3734d298b166.png)
